frontend/src/Editor/LeftSidebar/SidebarComment.jsx

```diff
@@ -7,15 +7,17 @@ export const LeftSidebarComment = ({ toggleComments, appVersionsId }) => {
   return (
     <LeftSidebarItem
       commentBadge
-      tip="toggle comments"
+      tip={appVersionsId ? 'toggle comments' : 'At least one version of the app should be deployed'}
       icon={`comments`}
       className={cx(`left-sidebar-item sidebar-zoom position-relative`, {
         disabled: !appVersionsId,
         active: isActive,
       })}
       onClick={() => {
-        toggleActive(!isActive);
-        toggleComments();
+        if (appVersionsId) {
```

Comment:
I feel this condition is not required here as its functionality is already achieved by `disabled: !appVersionsId` on line number 13.

Reply:
@sherfin94 If the app is disabled ,the tooltip will not show

Reply:
@saisumanthkumar Actually the click event works because of adding the pointer events css[here](https://github.com/ToolJet/ToolJet/pull/1359/files#diff-1ff9cafa7affbc78d3367c1918f48b0cce543785adfe5064010262f68c30e7c9R23)

Any particular reason to add that css property?

Reply:
@gondar00 yeah by default the pointer event is none , so even if we hover nothing happens , so I set it to visible

Reply:
![image](https://user-images.githubusercontent.com/12490590/140748706-0421702f-a2f2-4dcb-8b19-89580d4193c9.png)
https://github.com/react-bootstrap/react-bootstrap/issues/1588

Hey @saisumanthkumar, I have found few references around why the tooltip wasn't working on disabled elements, 

Instead of adding pointer events: all/visible; recommended way is to wrap the disabled element around a <div />

Reply:
@gondar00 thanks for the ref.